.github/workflows/mariner2.yml

```diff
@@ -0,0 +1,46 @@
+name: build for mariner2
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  build-mariner2:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l -o pipefail {0}
+    container:
+      image: mcr.microsoft.com/cbl-mariner/base/core:2.0
+    steps:
+      - name: install pre-checkout dependencies
+        run: tdnf install --noplugins --skipsignature -y ca-certificates git
+      - name: checkout
+        uses: actions/checkout@v4
+        with:
+          submodules: 'recursive'
+          set-safe-directory: true
```

Comment:
Can checkout get true here? I was under the impression it takes a path (or * for all).

Reply:
yes, when I tried with `"'*'"` (as I saw you did) I got an error...

Reply:
and without specifying `set-safe-directory` does it work?

Reply:
yes, because I have `git config --global --add safe.directory $PWD` in `mariner2.sh`